### ISSUE TITLE
update generic reponse

### ISSUE THREAD

**Issue by rosscortb** [SalesforceLabs/IssueBox#8]
Hello,

Is it possible to amend this message when the ticket is created - Thanks for reporting this issue. We will notify you when someone starts working on this issue.

Thanks
Ross

**Comment by jrattanpal** [comment]
These messages are in the Flow. You should be able to change it in Flow (or Copy that Flow and customize it)

https://github.com/SalesforceLabs/IssueBox/blob/master/force-app/main/default/flows/IssueBox.flow-meta.xml

**Comment by rosscortb** [comment]
@jrattanpal Thanks. The bell notification doesn't appear to be working. When I do test and log an issue as system administrator I don't receive a bell notification. I noticed there is a issue box process but it can't be viewed, is this something I need to build myself?

**Comment by jrattanpal** [comment]
@rosscortb 

As per instructions on https://github.com/SalesforceLabs/IssueBox, 

_However, customers can create their own process builder to @mention, say, a group of admins who will be responisble for these issues. Those admins will be notified when that issue is created. Customers can use packaged Process Builder to clone and make changes as needed_

You will need to create a new Flow to @mention the group. That is mainly because Chatter groups are data and cannot be packaged. So, Customers will need to create their own group (if needed) And add a flow .

Yes, we also recommend using Flow going forward. I will need to update this application to change my existing process build to Flow in newer version.

**Comment by jrattanpal** [comment]
I stand corrected in my earlier message.

Those messages are in process builder and that is not editable anymore. Process Builder will be EOL soon. 

For now, you can create your Flow to also move the existing features (@mentioning user etc) in Flow and change messages. You can deactivate process in the the process bulider.

(At least until I can release with flow. That may not be aavailble for a while due to my scehdule although I will try to release soon)

**Comment by jrattanpal** [comment]
@rosscortb 

I was able to update this today to verson 1.3. In this version, I have retired (deactivated) the process and moved that functionality to 2 Flows. I have marked those flows as template. You could leave those as-is or you can clone those flows and make changes as needed.

https://appexchange.salesforce.com/appxListingDetail?listingId=a0N3A00000FoU7IUAV

**Comment by rosscortb** [comment]
Hello,

Thanks. It's a great app.  Showed it to my boss so sorry for all the
questions on the odd tweak.

I upgraded earlier and its shows two apps in Sandbox, is that correct if so
is the latest one with standard after it ?-  Thanks Ross
[image: image.png]

**Comment by jrattanpal** [comment]
Yes, I added one yesterday. One is with console navigation and other one is standard navigation. I thought I should add that so there are both. 

Of course, you can create your app in Setup->App Manager if needed and drag/drop components from my managed pacakge.
Thanks

**Comment by rosscortb** [comment]
Ok, thanks, Sorry, one other question.
Worth noting I am using this app through Sage People built on Salesforce.

So, logging in as another user. They can add the Issue tab to their menu
bar.

[image: image.png]

They can click on New, but this option doesn't allow screenshot. It is
possible to have the utility bar at the bottom, as shown in the Issue app
page?

[image: image.png]

Thanks
Ross

**Comment by jrattanpal** [comment]
Images didn't show up in the message.  If you click Issues in the menu bar then that takes you to object. 

But if you open the app, where utility bar is added, it should be visible to the logged in user.
